recipes-support/libgpiod/libgpiod_%.bbappend

```diff
@@ -0,0 +1 @@
+PACKAGECONFIG_append = " python3"
```

Comment:
ok, I had to revisit the manual to double check enable/disabling PACKAGE CONFIG features, and their respective behavior through  four order-dependent arguments. In this branch we have `meta-openembedded/meta-oe/recipes-support/libgpiod/libgpiod_1.4.3.bb` with `PACKAGECONFIG[python3] = "--enable-bindings-python,--disable-bindings-python,python3"` when you enable this (with the bbappend `PACKAGECONFIG_append = " python3"` )  `--enable-bindings-python,` is added to `[EXTRA_OECONF]`and `python3` is added to  build dependencies. So that bbappend makes sense and should stay. We can lose the python3 being set as distro feature.

Reply:
Whoops previous comments were made before reading this one - agree with everything you said here.